SnailTests/ReplayTests.swift

```diff
@@ -0,0 +1,58 @@
+//  Copyright © 2016 Compass. All rights reserved.
+
+import Foundation
+import XCTest
+@testable import Snail
+
+class ReplayTests: XCTestCase {
+    private var subject: Replay<String>?
+
+    override func setUp() {
+        super.setUp()
+        subject = Replay<String>(2)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func testEvent() {
+        var strings: [String] = []
+        subject?.subscribe { event in
+            switch event {
+            case .next(let string):
+                strings.append(string)
+            default: break
+            }
+        }
+        subject?.on(.next("1"))
+        XCTAssert(strings.first == "1")
+    }
+
+    func testOnNext() {
+        var strings: [String] = []
+        subject?
+            .subscribe(onNext: { string in strings.append(string) })
+        subject?.on(.next("1"))
+        subject?.on(.next("2"))
```

Comment:
This isn't testing replay. It's just testing normal observables. You need to `on` before you subscribe, right?